.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "0.12"
```

Comment:
is it worth testing 0.12, or maybe instead test for v4 which is closer to the v5 extended life launch?